src/contexts/globalLayout/GlobalNavbar/GalleryNavbar/GalleryRightContent.tsx

```diff
@@ -110,13 +131,15 @@ export function GalleryRightContent({ queryRef, username }: GalleryRightContentP
     );
   }
 
-  if (shouldShowEditButton) {
+  if (showShowMultiGalleryButton) {
     return (
-      <EditButtonContainer onClick={handleEditClick}>
-        <TitleXS>EDIT</TitleXS>
-
-        {dropdown}
-      </EditButtonContainer>
+      <HStack gap={12}>
+        {editGalleryUrl && (
+          <Button variant="secondary" onClick={handleCreateGallery}>
+            Add New
+          </Button>
+        )}
+      </HStack>
```

Comment:
this code removes the "Edit" button from the top right when you're not on the `galleries` route.

<img width="1254" alt="image" src="https://user-images.githubusercontent.com/6754223/213228269-1a8ef38d-faf1-42ca-bc60-1341ea34315a.png">

Reply:
Fixed this!

Reply:
great catch